verifact/forum/schema.py

```diff
@@ -39,3 +40,87 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+# MUTATIONS
+
+
+class CreateQuestion(Mutation):
+    class Arguments:
+        created_at = DateTime()
+        text = String()
+        citation_url = String()
+        citation_title = String()
+        citation_image_url = String()
+
+    question = Field(QuestionNode)
+
+    def mutate(
+        self,
+        info,
+        text,
+        citation_url,
+        citation_title=None,
+        citation_image_url=None,
+        created_at=None,
+    ):
+        question = Question.objects.create(
+            created_at=created_at,
+            text=text,
+            citation_url=citation_url,
+            citation_title=citation_title
+            if citation_title is not None
+            else "Untitled",  # TODO: fetch with OpenGraph if not supplied, or in future more likely implement separate mutations to fetch OpenGraph metadata, then update with OpenGraph metadata if user is happy with it
+            citation_image_url=citation_image_url
+            if citation_image_url is not None
+            else "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Noun_Project_question_mark_icon_1101884_cc.svg/1024px-Noun_Project_question_mark_icon_1101884_cc.svg.png",  # TODO: same as 3 lines up, OpenGraph/metadata retrieval
```

Comment:
I think if the `citation_image_url` isn't present, the front-end can decide what to do. So leaving this blank would be my preference. It would be a bit weird to have this identical string saved for many rows when the front-end can just use logic to show a specific image. Plus it's more bits in the payload!

Reply:
ok - setting default to empty string rather than actual None as per https://docs.djangoproject.com/en/3.1/ref/models/fields/#null